### PR TITLE
SEO-175680-Windowsforms-UG-page-with-redirects-link

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: WindowsForms/GridGrouping/Grid-Layout.md [hunk 1]
```diff
@@ -838,7 +838,7 @@ N> To view a sample on Freezing the Caption Row, refer the dashboard sample whic
 ![Grid-Layout_img24](Grid-Layout_images/Grid-Layout_img24.png)
 
 ### Enabling ScrollBar when the Freeze Columns exceeds the Client Area
-In GridGroupingControl, if freeze columns exceed client area then the horizontal scroll bar will become invisible. To overcome this some of the customizations has to be done. You have to make use of the [ResizeEnd](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.form.resizeend?view=net-5.0) and [HScrollPixelPosChanging](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridControlBase.html) events and need to check certain conditions in it.
+In GridGroupingControl, if freeze columns exceed client area then the horizontal scroll bar will become invisible. To overcome this some of the customizations has to be done. You have to make use of the [ResizeEnd](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.form.resizeend?view=windowsdesktop-7.0&viewFallbackFrom=net-5.0) and [HScrollPixelPosChanging](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridControlBase.html) events and need to check certain conditions in it.
```
Comment (Yvone-Atieno) [inline WindowsForms/GridGrouping/Grid-Layout.md:841]: Update correct cr links

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/GridGrouping/Grid-Layout.md (hunk 1 above)
Comment (Edithabich) [reply WindowsForms/GridGrouping/Grid-Layout.md]: Resolved comments

File: WindowsForms/GridGrouping/Filtering.md [hunk 2]
```diff
@@ -86,12 +86,12 @@ Me.gridGroupingControl1.TableDescriptor.Columns(1).FilterRowOptions.AllowEmptyFi
 N> The removing of `Custom` and `Empty` text from the filter dropdown is also applicable for the [dynamic filter](#dynamic-filter).
 
 ## Record filters
-The [RecordFilters](http://help.syncfusion.com/cr/windowsforms/Syncfusion.Grouping.RecordFilterDescriptorCollection.html#) otherwise known as `RowFilters` will allow users to restrict displayed records to those that satisfy logical condition that user specify with `FilterRowDescriptor`.
+The [RecordFilters](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Grouping.RecordFilterDescriptorCollection.html) otherwise known as `RowFilters` will allow users to restrict displayed records to those that satisfy logical condition that user specify with `FilterRowDescriptor`.
```
Comment (Yvone-Atieno) [inline WindowsForms/GridGrouping/Filtering.md:89]: Update this link - https://help.syncfusion.com/cr/windowsforms/Syncfusion.Grouping.TableDescriptor.html#Syncfusion_Grouping_TableDescriptor_RecordFilters

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/GridGrouping/Filtering.md (hunk 2 above)
Comment (Edithabich) [reply WindowsForms/GridGrouping/Filtering.md:89]: udated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**